ripley.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+require 'yaml'
+
+
+def format_geojson_from_docs(docs)
+    out = {
+        type: 'FeatureCollections',
+        features: [],
+    }
+    docs.each {|doc|
+        coordinates = doc.delete('coordinates')
+        if coordinates
+            out[:features].push({
```

Comment:
you're going to have to show me this one. I can delete the curly braces, but when I delete the parens, I get a syntax error